src/frontend/test_runner/tests/testdata/tpch.yaml

```diff
@@ -1026,7 +1033,6 @@
         and l_shipmode in ('AIR', 'AIR REG')
         and l_shipinstruct = 'DELIVER IN PERSON'
       );
-  binder_error: 'Feature is not yet implemented: unsupported expression InList { expr: Identifier(Ident { value: "p_container", quote_style: None }), list: [Value(SingleQuotedString("SM CASE")), Value(SingleQuotedString("SM BOX")), Value(SingleQuotedString("SM PACK")), Value(SingleQuotedString("SM PKG"))], negated: false }, Tracking issue: https://github.com/singularity-data/risingwave/issues/112'
```

Comment:
try enabling logical / stream / batch plan for this case?

Reply:
Fixed